daemon/algod/api/algod.oas2.json

```diff
@@ -4441,6 +4497,33 @@
           }
         }
       }
+    },
+    "BlockLog": {
```

Comment:
Now that I understand the schema better (sorry about that), I think this object should not have `Block` in the name.  These are all of the logs for a single app-call, right? How about `AppCallLogs`?